PENDING_CHANGELOG.md

```diff
@@ -81,6 +81,7 @@ const DBTable = new dynamoose.Table("DBTable", [User]);
 
 - Fixed bug where using `model` as an attribute name in your Schema would cause random TypeMismatch errors.
 - Fixed an issue where after doing certain Item related operations, the original object passed in could be mutated. This is no longer the case and original items will not be mutated when doing Item related operations.
+- Fixed issue where sometimes indexes would be deleted due to `undefined` property values in the `describeTable` response.
```

Comment:
```suggestion
```

We actually don't need this. It didn't happen when running v2, and only started occurring after upgrading to v3.